config/environments/staging.rb

```diff
@@ -0,0 +1,90 @@
+Rails.application.configure do
```

Comment:
So we need to make sure that the staging site is behaving in the staging environment.